filters/include/pcl/filters/conditional_removal.h

```diff
@@ -38,8 +38,6 @@
 #pragma once
 
 #include <pcl/memory.h>
-#include <pcl/pcl_macros.h>
```

Comment:
you need to include pcl_config, because of `PCL_NO_PRECOMPILE` at the end of this file.